django-reference-app/django-reference-app/static/modelforms.py

```diff
@@ -0,0 +1,7 @@
+from django import forms
+from govuk_template_base.dbmodels import Person
+
+class tableForms(forms.ModelForm):
+    class Meta:
+        model= Person
+        fields= ["name","job"]
```

Comment:
Very minor point, but there should be a space between list items, as per [PEP8](https://www.python.org/dev/peps/pep-0008/).